package.json

```diff
@@ -10,8 +10,10 @@
         "dev": "pnpm run -r --filter=web-mapviewer dev ",
         "format": "prettier --write .",
         "format:check": "prettier --check .",
-        "lint": "eslint --fix",
-        "lint:no-fix": "eslint",
+        "lint": "eslint --fix && stylelint \"**/*.{vue,scss}\" --fix",
```

Comment:
I'm wondering whether we should use `||` instead. So that when the eslint fails, we still get the feedback from `styleint`